BE/sidedish11/src/main/java/dev/codesquad/java/sidedish11/entity/Badge.java

```diff
@@ -0,0 +1,43 @@
+package dev.codesquad.java.sidedish11.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.Id;
+
+import java.util.Objects;
+
+@Getter @Setter
```

Comment:
한 줄에 하나의 애노테이션을 선언해 주세요.

```suggestion
@Getter
@Setter
```